app/templates/core/create-account-modal/oz-vs-coco-view.jade

```diff
@@ -0,0 +1,34 @@
+button.back.back-button
```

Comment:
Can we switch to two-space indentation in this file?

Also, no need to change this one, but we would try to create new files with the `.pug` extension, as we're eventually wanting to migrate everything to that.

Reply:
@adamkecskes can u make Nick's requested change

Reply:
@mrfinch indentation changed to two-space.